app/services/support_interface/locations_export.rb

```diff
@@ -0,0 +1,52 @@
+module SupportInterface
+  class LocationsExport
+    def data_for_export
+      application_choices.map do |application_choice|
+        application_form = application_choice.application_form
+
+        {
+          'Candidate id' => application_form.candidate_id,
+          'Support reference' => application_form.support_reference,
+          'Age' => return_age(application_form),
+          'Candidates postcode' => application_form.postcode,
```

Comment:
Minor nit: Should this be _Candidate's postcode_ or maybe just _Candidate postcode_? Ditto for the next two lines.